Assets/MixedRealityToolkit/_Core/Definitions/Devices/MixedRealityInteractionMapping.cs

```diff
@@ -263,6 +263,30 @@ private set
             }
         }
 
+        private bool updated;
+
+        /// <summary>
+        /// Has the value been updated since the last reading?
```

Comment:
This summary (and the name `Updated`) doesn't seem accurate.

All the places that set this check for "changed or not the default value", which doesn't guarantee that the value has been updated since the last reading. It might be the same reading but non-default.